docker/generic/start_proxy.py

```diff
@@ -496,6 +496,18 @@ def make_argparser():
         the location of the service account credentials JSON file. If the option is
         omitted, the proxy contacts the metadata service to fetch an access token.
         '''.format(creds_key=GOOGLE_CREDS_KEY))
+
+    parser.add_argument(
+        '--dns_resolver_address',
+        help='''
+        The IP address or the IP address followed with port number used to
```

Comment:
maybe just:

IP address of the DNS resolver

Reply:
Use `address of DNS resolver` since users can assign PORT also.